### ISSUE TITLE
Embedded Session Example

### ISSUE THREAD

**Issue by jmcc0nn3ll** [eclipse/jetty.project#613]
We should have an example in our embedded examples that wires up the hashsession impl in front of a session using hello world servlet

**Comment by luisalduucin** [comment]
Hi @jmcc0nn3ll I would like to contribute with this issue. I think it is needed to do something like the testHashSession test in the HashSessionManagerTest class. Am I going in the right direction ? Thanks in advance.

**Comment by janbartel** [comment]
@luisalduucin  thanks for helping out.  

A simple example that extends the existing embedded example [One Context](https://github.com/eclipse/jetty.project/blob/jetty-9.3.x/examples/embedded/src/main/java/org/eclipse/jetty/embedded/OneContext.java) might be useful.

FYI, with jetty < 9.4, if you simply create a SessionHandler it will automatically create a HashSessionManager and HashSessionIdManager if no particular SessionManager and SessionIdManager are explicitly set up.  So the code could be as simple as:
```java
       ContextHandler context = new ContextHandler();
        context.setContextPath( "/hello" );
        SessionHandler s = new SessionHandler();
        s.setHandler(new HelloHandler());
        context.setHandler(s);
```

With jetty 9.4 the SessionHandler will also use some defaults if nothing is configured (DefaultSessionCache, NullSessionDataStore and DefaultSessionIdManager).

**Comment by luisalduucin** [comment]
Thanks for taking a moment to answer @janbartel .

I have already done an example basing on the the [One Servlet Context](https://github.com/eclipse/jetty.project/blob/jetty-9.3.x/examples/embedded/src/main/java/org/eclipse/jetty/embedded/OneServletContext.java) example.

The creation of the HashSessionManager and the HashSessionIdManager I did them explicit in order to be more explainable.

I am going to create a PR and if you think my example is not correct I would change it.

**Comment by blzb** [comment]
Cool, thanks for the example !

**Comment by janbartel** [comment]
This was merged into jetty-9.3
